showcase/app/styles/app.scss

```diff
@@ -13,6 +13,9 @@
 @import "./typography";
 @import "./globals";
 
+// this file comes from the 'ember-flight-icons' package
+@import "icons";
+
```

Comment:
question: we plan to remove this in #2338, right? if so, don't we temporarily need it in `website` too?

Reply:
We actually don't need it here anymore either after removing the foundations/icon page